README.md

```diff
@@ -96,6 +96,12 @@ pip3 install poetry
 poetry install
 poetry shell
 alembic upgrade head
+
+cd openadapt/app/dashboard
+nvm use
+npm install
+poetry install
```

Comment:
How can we automatically install this as part of the previous call to `poetry install`?

e.g. https://stackoverflow.com/a/69159741

Reply:
I have removed the `pyproject.toml` from the dashboard directory, since there were only two packages, both of which were already being used in the main project.